index.html

```diff
@@ -191,18 +191,21 @@ <h3>For argon2</h3>
                     try {
                         outputSpan.className = '';
                         var password = passwordInput.value;
-                        var locationSalt = UniquePasswordBuilder.getSaltOnLocation(urlInput.value);
-                        if(locationSalt !== '') {
-                            usefulUrl.textContent = 'Key used to generate password: ' + locationSalt;
-                        }
-                        else {
-                            usefulUrl.textContent = '';
-                        }
 
                         var result = UniquePasswordBuilder.verifyPassword(password);
                         if (!result.success) {
                             setErrorMessage(result.message, result.error);
                         } else {
+                            var locationSalt = UniquePasswordBuilder.getSaltOnLocation(urlInput.value);

```

Comment:
I think we need to check it on the addon too.

Reply:
Alright. I thought it was not very important because the url is filled with the one of the current tab but I will made it...

Reply:
You made a point but It’s better to unify behavior between addon and index page.